### PR TITLE
Modified code of DaoGen_Tool to replace the TimeStamp code when it has been selected.

### PR DESCRIPTION
**Pull request by SandeepNayak077**
1) Added TimeStamp placeholder's Key values in app.config file and
2) Added constant to get the TimeStamp placeholder from the app.config file and
3) Did code modification to replace the required TimeStamp code in the template, if TimeStamp selected in the tool.
4) Did modification in the DGenTemplates files(SearchAndUpdate.aspx.cs and SearchAndUpdate.aspx.vb)

### CODE REVIEW

File: root/programs/C#/Frameworks/Tools/DaoGen_Tool/app.config [hunk 1]
```diff
@@ -210,7 +212,7 @@
     <!--GridView Header Column Number-->
     <add key="RpColumnNmbr" value="_ColumnNmbr_"/>
     <!-- For Internationalization Test -->
-    <add key="DefaultCulture" value="ja-JP"/>
+    <add key="DefaultCulture" value=""/>
```
Comment (daisukenishino2) [inline root/programs/C#/Frameworks/Tools/DaoGen_Tool/app.config]: Does not change.

File: root/files/tools/DGenTemplates/SearchAndUpdate.aspx.vb [hunk 2]
```diff
@@ -235,7 +235,9 @@ Partial Public Class _TableName_SearchAndUpdate
         ' ControlComment:LoopEnd-PKColumn
 
         ' タイムスタンプ列
-        ' ・・・
+		' ControlComment:ControlComment:IsRequired-TimeStamp
+		' ...
+        parameterValue.AndEqualSearchConditions.Add("_TimeStampColName_", "")
```
Comment (daisukenishino2) [inline root/files/tools/DGenTemplates/SearchAndUpdate.aspx.vb]: If the time stamp column name is not specified
In that case, Is it implemented as this line not output?

File: root/files/tools/DGenTemplates/SearchAndUpdate.aspx.cs [hunk 3]
```diff
@@ -246,8 +246,10 @@ protected string UOC_btnBatUpd_Click(FxEventArgs fxEventArgs)
         // ControlComment:LoopEnd-PKColumn
 
         // タイムスタンプ列
-        // ・・・
-
+		// ControlComment:ControlComment:IsRequired-TimeStamp
+		// ...
+        parameterValue.AndEqualSearchConditions.Add("_TimeStampColName_", "");
```
Comment (daisukenishino2) [inline root/files/tools/DGenTemplates/SearchAndUpdate.aspx.cs]: If the time stamp column name is not specified
In that case, Is it implemented as this line not output?

**Comment by daisukenishino** [comment]
I understand that correspondence has been completed.
When you will corrected part of ja-jp, then I will merge the code. 
After the tutorial will be updated, please test by performing the tutorial.

File: root/programs/C#/Frameworks/Tools/DaoGen_Tool/app.config (hunk 1 above)
Comment (SandeepNayak077) [inline root/programs/C#/Frameworks/Tools/DaoGen_Tool/app.config]: Thank you Nishino-san for your valuable comments
HODC will change accordingly and deliver the code.

**Comment by SandeepNayak077** [comment]
Does the tutorial updated by HISOL?
After updating the tutorial document, HODC will perform the test as per the tutorial.

Please confirm

**Comment by daisukenishino** [comment]
Tutorial will be provided from HISOL during this month.

**Comment by SandeepNayak077** [comment]
Thank for the confirmation Nishino-san.

**Pull request closed without merging**